sirepo/server.py

```diff
@@ -251,7 +256,7 @@ def api_findByNameWithAuth(
             query=m.includeMode and PKDict(application_mode=application_mode),
         )
 
-    @sirepo.api.Spec("require_user")
+    @sirepo.api.Spec("require_user", filename="SimFileName", data="SimData all_input")
```

Comment:
data is not SimData. This is a weird method. call it ApplicationDataSpec (no need for all_input) or filename. That will be available from ApplicationDataSpec. Change from data to spec.

Reply:
9d450384bbf20294b6007b678c3dde0143022f40